CODEOWNERS

```diff
@@ -287,3 +287,8 @@ k8s/aat/common/unspec/unspec-service.yaml @hmcts/unspec-admins
 k8s/aat/common/unspec/identity.yaml @hmcts/unspec-admins
 k8s/demo/common/unspec/unspec-service.yaml @hmcts/unspec-admins
 k8s/demo/common/unspec/identity.yaml @hmcts/unspec-admins
+
+
+### Kustomization control
+
+k8s/namespaces/**/kustomization.yaml @hmcts/platform-engineering
```

Comment:
We take control of kustomization on all namespaces as adding here adds to all environments ( even prod)